exceptionally-large-transfer-processing-engine/src/quote-consumer.ts

```diff
@@ -0,0 +1,29 @@
+import * as kafka from 'kafka-node';
+import { RedisClient } from 'redis';
+import { log } from './helper';
+import { publish } from './producer';
+import { getILPList } from './redis-client';
+
+const handleQuoteMessage = async (
+  message: kafka.Message,
+  topic: string,
+  redisClient: RedisClient,
+) => {
+  const jMessage = JSON.parse(message.value.toString());
+  const { quoteId } = jMessage;
+  await log(`Handling message with quote ID ${quoteId}`, topic);
```

Comment:
Don't think we have to await this? It can happen whenever. Even  wondering if we should be doing these logs? Wonder how big a hit it is on processing. Leave it for now I think, then if we go into performance check sprint, we can monitor this.